src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_button.scss

```diff
@@ -68,32 +67,47 @@ input[type="reset"],
 input[type="submit"],
 input[type="button"] {
     @include button-base;
-    @include button-color($color-text-highlight-normal, $color-text-inverted);
-
-    $border-width: 0.25em;
-
-    border: $border-width solid;
-    font-weight: $font-weight-extrovert;
+    $button-height: 24px / $font-size-normal;
+    background: transparent;
+    border: 0;
+    color: $color-text-highlight-introvert;
+    font-size: $font-size-normal;
+    font-weight: $font-weight-normal;
+    line-height: $button-height;
+    height: $button-height * 1em;
+    padding: 0;
     text-transform: uppercase;
+    min-width: 1px;
```

Comment:
I think this can be removed as it is no longer needed to overwrite `min-width: 9em` from line 79.